src/detectors/BEMC/CalorimeterHit_factory_EcalBarrelSciGlassRecHits.h

```diff
@@ -40,7 +40,7 @@ class CalorimeterHit_factory_EcalBarrelSciGlassRecHits : public JFactoryT<edm4ei
 
         // geometry service to get ids, ignored if no names provided
         m_geoSvcName="geoServiceName";
-        m_readout="EcalBarrelHits";  // from ATHENA's reconstruction.py
+        m_readout="EcalBarrelSciGlassHits";  // from ATHENA's reconstruction.py
         m_layerField="";              // from ATHENA's reconstruction.py (i.e. not defined there)
         m_sectorField="sector";       // from ATHENA's reconstruction.py
```

Comment:
From what I understand, juggler required same names for SciGlass as for Imaging, even though the former doesn't have layers. It's nice that this has no reference to layers.

I'm looking to fix the readout naming in https://github.com/eic/epic/pull/188, but that would require figuring out juggler benchmarks too.

Reply:
Alright. Sounds good. I'm going go ahead and merge this into main so I can tag a new release now. Not really good practice merging my own PRs, but tight timelines motivate quick action.